scipy/spatial/ckdtree/src/ckdtree_globals.cxx

```diff
@@ -0,0 +1,7 @@
+#define NPY_NO_DEPRECATED_API NPY_API_VERSION
```

Comment:
I see there's a similar issue in the sgemv fix by the way, I'll send a PR for that (also for 0.15.1).